docker-compose.yml

```diff
@@ -29,9 +23,10 @@ services:
       DEPENDENCIES: "influxdb"
     labels:
       - "ServiceUUId=system.telegraf"
+      - "io.appcelerator.amp.build.mounts=/var/run/docker.sock,/var/run/utmp"
```

Comment:
Not currently. But I could certainly add a way.  I'd like to support everything docker services support.

On Jul 22, 2016 5:14 AM, ndegory <notifications@github.com> wrote:

In docker-compose.yml<https://github.com/appcelerator/docker-influxdb/pull/14#discussion_r71868993>:

> @@ -29,9 +23,10 @@ services:
>        DEPENDENCIES: "influxdb"
>      labels:
>        - "ServiceUUId=system.telegraf"
> +      - "io.appcelerator.amp.build.mounts=/var/run/docker.sock,/var/run/utmp"


got it! is there a way to specify the mount as read-only?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/appcelerator/docker-influxdb/pull/14/files/b9e1982ee0e78156bae9ce9e20172b703ffdbc4d#r71868993>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AAcFqPPOPVZfX1C335mlyc7y5ADMbXt_ks5qYLPugaJpZM4JSQaN>.